app/views/snps/_snpedia_papers.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="snpedia">
+  <h3>Links to SNPedia</h3>
+  <% if snp.snpedia_papers.present? %>
+    <table class="table table-striped">
+      <tr>
+        <th>Title</th>
+        <th>Summary</th>
+      </tr>
+      <% snp.snpedia_papers.each do |p| %>
+        <tr>
+          <td>
+            <%= bold_if_matching_local_genotype(p, user_snp) do %>
+              <%= link_to "#{snp.name} #{p.local_genotype.split('').join('/')}", p.url %>
+            <% end %>
+          </td>
+          <td><b><%= p.summary.capitalize %></b></td>
+        </tr>
+      <% end %>
+    </table>
+  <% else %>
+    There are no SNPedia-links yet. Please come back later.
+  <% end %>
+  <div class="small">
+    Note: For some SNPs, testing providers detect the genotype from the opposite strand of DNA, so that the genotypes listed at openSNP don't appear here. In those cases, please replace "A" by "T" or "G" by "C" (and vice-versa).
```

Comment:
Ah, okay. So we're talking about actually a biological strand flip here. Not the ordering of the two alleles in a genotype. 

Whether it's "GC" or "CG" is indeed irrelevant for this and we already should check for this at this point w/ SNPedia iirc.

But: You know that C <-> G and T <-> A pair in the double stranded DNA. So when 23andMe or other companies read out your SNPs it actually matters which of the two strands they are looking at. In the easy case you can have 3 genotypes at a given SNP, let's say AA, AG or GG. If you now have a user who is TC, TT or CC you can directly infer that the company reported the opposite strand compared to SNPedia. 

With me so far? Okay, now imagine the genotypes are CC, CG or GG. In that case you have no way of figuring out whether the genotype you are looking is, is from the correct strand. 

Does this make more sense? :-)